Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -1219,11 +1219,300 @@ bool GeomUtils::checkMeshSet( const carve::mesh::MeshSet<3>* mesh_set,
 	}
 #endif
 
-	if( err.tellp() > 0 )
+	if (err.tellp() > 0)
 	{
 		throw oip::InconsistentModellingException("GeomUtils::checkMeshSet(): MeshSet of resulting mesh has problems: " + err.str());
 	}
 	return true;
 }
 
+	/**********************************************************************************************/
+
+	/*!Calculate recursive multiplication for calculation Taylor series
+	* \param[in]  value                  value of Taylor series
+	* \param[in]  polynomialConstants    polynomial constants for a polyomial term - a,b,c,d
+	* \param[in]  polynomialConstantCnt  type of polinomial term - cubic, linear 
+	* \param[in]  myArray
+	* \param[in]  myArrayParentIndex
+	* \param[in]  i_n
+	* \param[in]  n
+	*
+	* \return
+	*/
+	void GeomUtils::recursiveMultiplicationTaylor(
+			double	value,
+			std::vector<double> polynomialConstants,
+			size_t	polynomialConstantCnt,
```

Comment:
instead of  ´polynomialConstantCnt` you can use `polynomialConstants.size()` - thus reducing the amount of variables to this function.

Reply:
Resolve 5442d2d